flopy/mf6/utils/codegen/templates/package.py.jinja

```diff
@@ -14,18 +14,18 @@ class Modflow{{ title }}(MFPackage):
 
     Parameters
     ----------
-    {{ macros.vars_docs(vars, indent=4) }}
+{{ macros.vars_docs(vars, indent=4) }}
     """
 
     {% for attr in vars|attrs %}
     {{ attr }}
-    {%- endfor %}
+    {% endfor %}
 
     def __init__(
         self,
         {{ name|parent }},
         loading_package=False,
-        {{ macros.init_vars(vars, indent=8, skip=name|skip_init) }}
+{{ macros.init_vars(vars, indent=8, skip=name|skip_init) }}
```

Comment:
```suggestion
        {{ macros.init_vars(vars, skip=name|skip_init)|indent(8) }}
```

I played around with the indentation. It is possible to apply a filter after a macro.
So I tried this in package.py.jinja:

```jinja
class Modflow{{ title }}(MFPackage):
    """
    {{ name|description }}

    Parameters
    ----------
    {{ macros.vars_docs(vars, indent=0)|indent(4) }}
    """
```

I had to add the 4 spaces in the front, because indent only applies to new lines.
But this also works:

```jinja
class Modflow{{ title }}(MFPackage):
    """
    {{ name|description }}

    Parameters
    ----------
{{ macros.vars_docs(vars, indent=0)|indent(4, first=true) }}
    """
```

See https://jinja.palletsprojects.com/en/stable/templates/#jinja-filters.indent

Reply:
This works very nicely